gulp/gulp/gulpfile.js

```diff
@@ -0,0 +1,60 @@
+const gulp = require('gulp');
+const sass = require('gulp-sass')(require('sass'));
+const browserSync = require('browser-sync').create();
+const concat = require('gulp-concat');
+const autoprefixer = require('gulp-autoprefixer');
+const zip = require('gulp-zip');
+const uglify = require('gulp-uglify');
+
+function liveReload() {
+  browserSync.init({
+    server: './src'
+  });
+
+  gulp.watch('src/sass/*.scss', cssTranspile);
+  gulp.watch('src/scripts/*.js', jsCompile);
+  gulp.watch("src/*.html").on('change', browserSync.reload);
+  gulp.watch('src/sass/*.scss', autoPref);
+  gulp.watch('src/zip/*', addZip);
+}
+
+function cssTranspile() {
+  return gulp.src('./src/sass/main.scss')
+    .pipe(sass().on('error', sass.logError))
+    .pipe(gulp.dest('./src/css'));
+}
+
+function jsCompile() {
+  return gulp.src('./src/scripts/*.js')
+    .pipe(concat('script.js'))
+    .pipe(gulp.dest('./src/js'));
+}
+
+function autoPref() {
+  return gulp.src('./src/sass/main.scss')
+    .pipe(autoprefixer({
+      cascade: false
+    }))
+    .pipe(gulp.dest('dist'))
```

Comment:
шлях до фінального файлу не зовсім правильний, потрібно помістити в папку css
```suggestion
    .pipe(gulp.dest('dist/css'))
```